docker-compose.yml

```diff
@@ -1,20 +1,35 @@
 version: '2'
 services:
-  app:
+  chokidar:
```

Comment:
Is there a reason we should call it this rather then just `app`?

Reply:
I'm open to just calling this `app`, I don't have a really strong opinion. My thinking was that it is a bit nicer in the output to actually know what the service was. Maybe we should call this `jekyll`?